inference-engine/src/gna_plugin/gna_model_serial.hpp

```diff
@@ -27,21 +27,21 @@
  */
 #if GNA_LIB_VER == 2
 #define HEADER_MAJOR 2
-#define HEADER_MINOR 1
+#define HEADER_MINOR 2
```

Comment:
Does this mean we are changing the version once again, so the files exported by 2020.4 will no longer be supported?

Reply:
Yes